packages/webpack-cli/lib/groups/HelpGroup.js

```diff
@@ -39,7 +39,16 @@ class HelpGroup {
         process.stdout.write('\n                  Made with ♥️  by the webpack team \n');
     }
 
-    outputVersion() {
+    outputVersion(externalPkg) {
+        if (externalPkg && externalPkg.name) {
+            try {
+                const { name, version } = require(`@webpack-cli/${externalPkg.name}/package.json`);
```

Comment:
What about if package will be out of webpack-cli repo, yep they doesn't exists right nowm but it is possible

Reply:
What do you think ? What should be done in that case?

Reply:
We should have original package name when we run external package

Reply:
Got it. I will update the logic accordingly.